code/modules/antagonists/abductor/equipment/glands/mindshock.dm

```diff
@@ -41,7 +41,7 @@
 		if(target_human.stat)
 			continue
 
-		if(HAS_TRAIT(target_human, TRAIT_MINDSHIELD))
+		if(HAS_MIND_TRAIT(target_human, TRAIT_UNCONVERTABLE))
```

Comment:
check for both

Reply:
Why was this marked as resolved, it doesn't seem to be done. I also don't think it should check for unconvertable at all?

Reply:
Well, mindshock has two effects: when the organ itself activates, it can cause hallucinations, confusions, and a brief stun to people nearby. You're right that this one should probably check for `TRAIT_MINDSHIELD` instead, and I've changed it now. But the other effect allows abductors to temporarily mind control people near the organ. Given that mechanically it seems like a temporary brainwashing, it seems like it should check for `TRAIT_UNCONVERTABLE` like brainwashing surgery does.